### PR TITLE
Changes to docs to reflect current behavior with respect to licensing

### PR DESCRIPTION
**Pull request by Bslabe123**
# Description

In [this PR](https://github.com/solo-io/solo-projects/pull/4641), we bumped the licensing repo and changed some of the behavior with respect to licensing. Prior to that we also made some changes that ensure Gloo doesn't crash with a missing license key. Our docs should reflect this new behavior.

# Checklist:

- [x] I included a concise, user-facing changelog (for details, see https://github.com/solo-io/go-utils/tree/master/changelogutils) which references the issue that is resolved.
- [x] If I updated APIs (our protos) or helm values, I ran `make -B install-go-tools generated-code` to ensure there will be no code diff
- [x] I followed guidelines laid out in the Gloo Edge [contribution guide](https://docs.solo.io/gloo-edge/latest/contributing/)
- [x] I opened a draft PR or added the work in progress label if my PR is not ready for review
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] I have added tests that prove my fix is effective or that my feature works

BOT NOTES: 
resolves https://github.com/solo-io/gloo/issues/7990

### CODE REVIEW

File: docs/content/operations/updating_license/_index.md [hunk 1]
```diff
@@ -14,48 +14,13 @@ When you first install Gloo Edge in your cluster, confirm the license key expira
 
 ## Confirm that your license expired
 
-Whether you're a prospective user using a trial license or a full Gloo Edge subscriber, this license can expire. When it does, you may see certain Gloo Edge pods start to display errors that are new to you.
+Whether you're a prospective user using a trial license or a full Gloo Edge subscriber, this license can expire. When it does, you may see warnings on your Gloo Edge deployments that are new to you.
```
Comment (jenshu) [inline docs/content/operations/updating_license/_index.md:17]: can you double check this? i believe some pods still do crash (e.g. observability, gloo-fed...) whereas some will work fine but log a warning

File: docs/content/operations/updating_license/_index.md [hunk 2]
```diff
@@ -102,3 +67,14 @@ Taking a fresh look at the k9s console shows us that all three of the failing po
 ![k9s Display with Refreshed License]({{% versioned_link_path fromRoot="/img/k9s-license-refreshed.png" %}})
 
 Congratulations! You have successfully replaced your Gloo Edge Enterprise license key.
+
+## Outdated License format
```
Comment (Rachael-Graham) [inline docs/content/operations/updating_license/_index.md:71]: Should this section be named something more like "Expired GraphQL license add-on"?

File: docs/content/operations/updating_license/_index.md [hunk 3]
```diff
@@ -102,3 +67,14 @@ Taking a fresh look at the k9s console shows us that all three of the failing po
...
+
+In some cases, Gloo Edge users who are have GraphQL features enabled may see the following message
```
Comment (Rachael-Graham) [inline docs/content/operations/updating_license/_index.md:73]:
```suggestion
If you enabled GraphQL features, you might see the following log on the `observability` deployment:
```

File: docs/content/operations/updating_license/_index.md [hunk 4]
```diff
@@ -102,3 +67,14 @@ Taking a fresh look at the k9s console shows us that all three of the failing po
...
+
+```bash
+% kubectl logs deploy/observability -n gloo-system
+{"level":"warn","ts":"2023-03-23T18:56:48.554Z","logger":"observability","caller":"client/client.go:195","msg":"Your gloo license graphql addon is outdated. Please contact support to update your license.","version":"1.14.0-beta10"}
+```
+
+If this message appears please contact support to get an up-to-date license key.
```
Comment (Rachael-Graham) [inline docs/content/operations/updating_license/_index.md:80]:
```suggestion
In this case, contact support to get a new license key.
```

File: docs/content/operations/updating_license/_index.md [hunk 5]
```diff
@@ -14,48 +14,13 @@ When you first install Gloo Edge in your cluster, confirm the license key expira
 
 ## Confirm that your license expired
 
-Whether you're a prospective user using a trial license or a full Gloo Edge subscriber, this license can expire. When it does, you may see certain Gloo Edge pods start to display errors that are new to you.
+Whether you're a prospective user using a trial license or a full Gloo Edge subscriber, this license can expire. When it does, you may see warnings on your Gloo Edge deployments that are new to you.
 
-For example, from the following [k9s](https://k9scli.io/) display, you can see that certain `gloo-system` pods fall into a `CrashLoopBackoff` state.
-
-![k9s Display with Expired License]({{% versioned_link_path fromRoot="/img/k9s-license-expired.png" %}})
```
Comment (jenshu) [inline docs/content/operations/updating_license/_index.md:21]: we can also delete the image `k9s-license-expired.png` from the repo

**Review by Rachael-Graham — APPROVED** [review]

File: docs/content/operations/updating_license/_index.md [hunk 6]
```diff
@@ -102,3 +67,14 @@ Taking a fresh look at the k9s console shows us that all three of the failing po
 ![k9s Display with Refreshed License]({{% versioned_link_path fromRoot="/img/k9s-license-refreshed.png" %}})
 
 Congratulations! You have successfully replaced your Gloo Edge Enterprise license key.
+
+## Expired GraphQL license add-on
```
Comment (jenshu) [inline docs/content/operations/updating_license/_index.md:71]:
```suggestion
## Older format licenses
```
this is not exactly referring to expired licenses, but rather licenses that were created in an older format (but are still valid)

Comment (jenshu) [reply docs/content/operations/updating_license/_index.md:71]: not sure that a good title for this section is, maybe "deprecated license format"?

File: docs/content/operations/updating_license/_index.md [hunk 7]
```diff
@@ -102,3 +67,14 @@ Taking a fresh look at the k9s console shows us that all three of the failing po
...
+
+If you enabled GraphQL features, you might see the following log on the `observability` deployment:
+
+```bash
+% kubectl logs deploy/observability -n gloo-system
+{"level":"warn","ts":"2023-03-23T18:56:48.554Z","logger":"observability","caller":"client/client.go:195","msg":"Your gloo license graphql addon is outdated. Please contact support to update your license.","version":"1.14.0-beta10"}
+```
+
+In this case, contact support to get a new license key.
```
Comment (jenshu) [inline docs/content/operations/updating_license/_index.md:80]:
```suggestion
Please note that Gloo Edge will still function correctly, as we accept both the new and deprecated license formats. However, to get rid of the warning in the logs, you should contact support to get a new license key.
```

**Review by jenshu — APPROVED** [review]

**Review by Rachael-Graham — APPROVED** [review]

**Review by sam-heilbron — COMMENTED** [review]
In this PR I see that we delete the reference to the license-expired image:
```
![k9s Display with Expired License]({{% versioned_link_path fromRoot="/img/k9s-license-expired.png" %}})
```

But we still have the image checked into the repo. Is this used elsewhere or can we delete the image itself?


UPDATE: I was confusing the expired and refresh images. Ignore me

**Pull request merged**